lib/html-proofer/check.rb

```diff
@@ -22,7 +23,9 @@ def run
     end
 
     def add_issue(desc, line: nil, status: -1, content: nil)
-      @issues << Issue.new(@path, desc, line: line, status: status, content: content)
+      @issues << Issue.new(@path, desc, "#{self::class } failed", line: line, status: status, content: content)
+      @stat.findings.push(Issue.new(@path, desc, "#{self::class } failed", line: line, status: status, content: content))
+      @stat.print_finding if @options[:stat]
```

Comment:
Doesn't this mean that on each issue, a new statement will be printed? Wouldn't it be preferable to provide that information at the end of all the runs? In other words, instead of iterating and printing `@issues`, it seems to make more sense to do something like:

```
@issues.each do |issue|
  @stat.print(issue)
```

Where `@stat.print` massages the `issue` format to whatever it likes.